### ISSUE TITLE
Instabug IOS SDK script variable/function's names collides with webview's URL script

### ISSUE THREAD

**Issue by ketan0809** [Instabug/Instabug-iOS#491]
### Steps to Reproduce the Problem
1. Open any URL in the webview.
2. Instabug iOS SDK is inserted into the webview's window.
3. This insertion collides with the webview's URL script.

### Expected Behavior
The Instabug SDK should not collide with other script's variables or function's names. 

### Actual Behavior
It collides with the webview's URL script variables or function's names.

### Possible solution
it is always recommended to wrap injected Instabug IOS scripts in [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE).

```
Original script:
 
"use strict";
    let _e,
        _t,
        _n,
        _i,
...
 
Fixed script:
 
(function () {
  "use strict";
    let _e,
        _t,
        _n,
        _i,
...
})();

```

### SDK Version
\>= 12.7.0 sdk version. 

### iOS Version
Any version

### Device Model
Any device

**Comment by krevis-figma** [comment]
Same problem here. I assume this is “Adds support for WebViews loading in App Performance Monitoring” in 12.7.0 ([release notes](https://github.com/Instabug/Instabug-iOS/releases)). Is there a way to turn this off?

**Comment by krevis-figma** [comment]
Aha, there is. After `Instabug.start()`, do this:
```
    APM.webViewTrackingEnabled = false
```

Side note: the documentation for this property is wrong:

```
/**
Controls whether tracking is enabled for web views in the application.

If set to `YES`, the application will enable tracking for web views, allowing for monitoring and recording of web vitals. If set to `NO`, tracking will be disabled.

The default value is `NO`.

@discussion Use this property to dynamically enable or disable web view tracking based on application requirements.
*/
@property (class, atomic, assign) BOOL webViewTrackingEnabled;
```

The default value is apparently actually `YES`.

**Comment by husseinkishk** [comment]
Hello @ketan0809 @krevis-figma 
We are sorry about this, we're working on a fix for this and it will be released soon.

**Comment by ketan0809** [comment]
@husseinkishk May I know, is it fixed in the v13.0.0 release?

**Comment by AhmedAshraf605** [comment]
@ketan0809 Yes, this is fixed in 13.0.0
